.circleci/config.yml

```diff
@@ -28,6 +37,7 @@ jobs:
 workflows:
   circleci:
     jobs:
+      - lint
       - test:
           filters:
             tags:
```

Comment:
We should require lint on release as well (line #47, but strangely I can't add a note to that line specifically.